aws-codegurureviewer-repositoryassociation/pom.xml

```diff
@@ -65,6 +65,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>codegurureviewer</artifactId>
+            <version>RELEASE</version>
```

Comment:
why RELEASE?

Reply:
The dependencyManagement specifies the version of software.amazon.awssdk to be 2.15.0. The tag feature of codegurureviewer is released in version 2.15.33.

Reply:
lets pick a version that's latest, i'm not sure if RELEASE picks the latest version.